pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -22,7 +22,7 @@ import (
 	"net/url"
 	"sort"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
I don't think this change is necessary

Reply:
This is a side effect of using `gofmt` (or `goexports`) and it's sort of inevitable as golang changes the formatting rules from time to time. Personally, I consider this is a price we have to pay for no coding style wars.

Reply:
ah, didn't realize it was linter driven :smile:

Reply:
Yeah, came from there indeed but I have reverted it :)